### ISSUE TITLE
make the jspackage folder configurable

### ISSUE THREAD

**Issue by ghost** [pontehu/jspm-mvn#1]
In some constellations it would be desirable to have the scripts under a different folder within the jar file. For example when using Servlet 3.0 with fragments, JavaScripts could be provided within the jar file in the folder `META-INF/resources`. If this could also be the path for jspm, one could use such jar files in both scenarios
* as jspm dependency
* as web resource directly in a Java Servlet Application

I tried to just put `package.json` and the `main.js` entry point into the directory `jspackage` and then from main.js define the exports relative

```
define(["exports", "../META-INF/resources/const/hello"], function(exports, hello) {
	exports.getHello = function() {
		return hello.hello;
	};
});
```
However, in the jspm case the META-INF folder will not be available after the repository was processing the jar and putting into the `jspm_packages` folder.

**Comment by zenorbi** [comment]
Would a global config param solve your problem, so you can configure jspm-mvn to **always** use `my/path/inside/jar` and not the default `jspackage`? I can provide that quickly. A bit harder would be the ability to define this in a per-jar basis.

**Comment by ghost** [comment]
This would fit my needs perfectly! :-)

**Comment by zenorbi** [comment]
Fixed in 822248b618f3bc1174cd53c411b47d4c60901a44 and published as 0.6.2.

You can configure the path using `jspm registry config mvn`

The second question should ask for the package path. Feel free to reopen this issue or open a new one if you have any further problems or questions.
